odoo/addons/base/models/assetsbundle.py

```diff
@@ -277,16 +284,17 @@ def clean_attachments(self, extension):
         must exclude the current bundle.
         """
         ira = self.env['ir.attachment']
+        is_css = extension in ['css', 'min.css', 'css.map']
```

Comment:
We may check this change of behaviour @Arcasias 
before that, the css.map was not ltr/rtl spécific.